OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -69,6 +70,12 @@ sub new ($class, @args) {
     return $self;
 }
 
+my $backend;
+
+sub create_backend ($self) { $backend = OpenQA::Isotovideo::Backend->new }
+sub backend ($self) { $backend }
```

Comment:
Wouldn't it make more sense to store the backend as a member variable and return it like this?
```suggestion
sub backend ($self) { $self->{_backend} }
```
Otherwise it is rather weird that this function randomly creates the last backend created by an arbitrary instance.

Reply:
Yes, but I wrote "**For now** we make it a singleton".
In #2216 it was done as a member variable, but I wanted to make **small** steps, like I said, because we had this regression which only one person on only one architecture only in cloud appeared.
In a **next** step I would move things further, and then it would be possible to make it a member variable.

Reply:
and regarding

> creates the last backend created by an arbitrary instance

There will only be one instance anyway.
I made this clear now by using `$backend ||=`.

It's the command handler, there is only one instance of it, and OpenQA::Isotovideo::Backend is a wrapper around `$bmwqemu::backend` (a global variable) anyway, so there can also be only one per process.

Reply:
With my current change I *can't* make it a member variable, because there is no instance of CommandHandler yet.
That's planned as one of the next steps.

Reply:
I renamed `$self` to `$class` now to make it clear.

Reply:
I think that should suffice.